arma3-windows/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM mcr.microsoft.com/windows/servercore:ltsc2022
+
+MAINTAINER mattis.kraemer (at) einfach-gaming.de
+
+RUN NET USER /add arma3
+RUN NET LOCALGROUP "Administrators" "arma3" /add

```

Comment:
Is it required that the arma3 user is an admin? I would prefer running the server with least privileges.

Reply:
I need to test this, might be possible to remove admin permissions.
But I might be forced to change the volume location

Reply:
So after testing we have permission problems with every part of the entrypoint and dockerfile afterwards. We could probably do it anyways but as everybody knows, the windows permission system is horrible to work with

Reply:
Do you run an ARMA server as Administrator normally on Windows? (e.g. using TADST).
If that's not the case, it should be possible to make it run without admin privileges. Maybe it's just about choosing the right folder where the data is placed.